helm/choreo-connect/values.yaml

```diff
@@ -148,6 +148,9 @@ wso2:
       # Define templated log_config.toml file, if empty using default log_config.toml
       logConfigToml: ""
 
+      # Mount API artifacts directory if the directory is empty
+      apiArtifactsMountEmptyDir: false
```

Comment:
Default should be true right? @renuka-fernando

Reply:
yes

Reply:
fixed

Reply:
@pubudu538 @renuka-fernando If the default config is `true`, do we need to add `RUN mkdir -p /home/wso2/artifacts/apis` to the Dockerfile of the adaper ?

Reply:
We will be adding this `RUN mkdir -p /home/wso2/artifacts/apis` in the next release of Choreo Connect. In Helm release 1.2.0.0 and these changes will be reverted.
Created an issue to track this https://github.com/wso2/kubernetes-microgateway/issues/61